Training4/src/task1/src/App.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import "./App.css";
+import {throttle} from 'lodash';
+
+function App() {
+  const [joke, setJoke] = useState("");
+  const controlClick= throttle(()=>getJoke(), 8000);
```

Comment:
controlClick create new every render, so you should wrap the controlClick with useCallback, because we don't want create controlClick many time and use it can make the throttle work incorrecta